### DISCUSSION TITLE
【读书会】《指数基金投资指南》 - 第1期 | ApacheCN

### DISCUSSION THREAD

**Discussion by yanqiang921106** [apachecn/home#234] [Ideas]
# <center>【读书会】《指数基金投资指南》 - 第1期 | ApacheCN<center>

* 【推荐】读书会: 《指数基金投资指南》银行螺丝钉
* 【推荐】留言地址: https://github.com/apachecn/home/issues/195

> 参与

* QQ群【临时群】: 1062944139
* 负责人紫色木槿: 1076003661 (QQ)

> 时间

**每两周三晚8点，月初会发布本月阅读书籍2-4本，顺序阅读。

* 时间周期: 2021-06-16 ～ 2021-06-30 （共 14 天，对自己严格的建议控制在7天内）
* 语音频率: 每两周三晚9点语音一次，周末休息跳过
* **群友发言：**5~10人，5分钟/人，计划时长0.5-1小时，带着问题发言	
* **[@紫色木槿](https://github.com/yanqiang921106)** 会进行语音: 沟通进度和督促学习

> 内容

1. 分享读书心得
2. 学到的知识
3. 可以应用到实际工作生活中的
4. 自己的应用计划			

每一本书，专门一个issues，供大家留言分享		

> 活动目的

1. 培养自己对理财的正确认识
2. 选出2个人作为大肥羊学校的学员，作为其他活动的代理班长		

---

* 总结：一人可以走的很快，但是一群人却可以走的很远

**Comment by yanqiang921106** [comment]
1、使用盈利收益率来定投指数基金的策略是：
 
·当盈利收益率大于10%时，分批投资。
·盈利收益率小于10%，但大于6.4%时，坚定持有已经买入的基金份额。
·当盈利收益率小于6.4%时，分批卖出基金。

2、盈利收益率适用条件：
1.流通性比较好、盈利比较稳定的品种
2.如果是盈利增长速度较快，或者盈利波动比较大的指数基金，则不适合使用盈利收益率法。

3、适合盈利收益率的品种：
上证红利、中证红利、上证50、基本面50、上证50AH优选、央视50、恒生指数和恒生中国企业指数（恒生指数、H股指数打9折作参考）

**Comment by yanqiang921106** [comment]
博格公式：P = E * PE
指数基金未来的年复合收益率，等于指数基金的投资初期股息率，加上指数基金每年的市盈率变化率，再加上指数基金的每年的盈利变化率。

博格公式应用：
1·在股息率高的时候买入。
2·在市盈率处于历史较低位置时买入。（以上这两点往往是同时发生的。）
3·买入之后，耐心等待“均值回归”，即等待市盈率从低到高。

**Comment by yanqiang921106** [comment]
博格公式拓展：P = B * PB
适用于盈利处于不稳定状态或呈周期性变化，但行业没有长期亏损记录的指数

**Comment by yanqiang921106** [comment]
![image](https://user-images.githubusercontent.com/34933285/121353167-22092480-c960-11eb-8882-c8539fb090e5.png)

**Comment by jiangzhonglian** [comment]
## 找鹅

> 货币基金(低风险、低收益)

* 企业债、银行定期存款、地方政府债、国债
* 风险+收益: 2%-4%
* 投资时间: 1年以内的短期债券
* 没有申购和赎回费用，这是和其他类型基金的一个典型区别
* 债券基金(低风险、低收益)

> 企业债、股票、地方政府债、国债

* 风险+收益: 6%-7%
* 投资时间: 1年以上的债券
* 投资债券占比 >= 80% + 其他占比 <= 20%

> 股票型基金(高风险、高收益)

* 风险+收益: 15%-20%，甚至更高
* 投资股票占比 >= 80% + 其他占比 <= 20%

> 混合型基金(高风险、高收益)

* 风险+收益: 15%-20%，甚至更高
* 基金经理调整投资占比: 偏股型基金 + 偏债型基金 + 平衡性基金

## 养鹅

* 余额宝： 2～3%
* 我要稳稳的幸福： 6～8%
* 沪深300、沪深300增强、白酒

## 瘦鹅

* 物美价廉的鹅：低估消费品（食品、消费、日用、饮料、白酒）

**Comment by yanqiang921106** [comment]
![image](https://user-images.githubusercontent.com/34933285/122882173-7eac0c80-d36e-11eb-9307-c40e3032b98e.png)

**Comment by jiangzhonglian** [comment]
1
